tests/test_rule_in_container.sh

```diff
@@ -248,7 +246,7 @@ if test -n "$ADDITIONAL_SSGTS_OPTIONS"; then
 	ADDITIONAL_TEST_OPTIONS="$ADDITIONAL_SSGTS_OPTIONS"
 fi
 
-command=(python3 "${script_dir}/automatus.py" rule ${ADDITIONAL_TEST_OPTIONS} --remove-machine-only --remove-ocp4-only "${additional_args[@]}" --add-platform "$test_image_cpe_product" "$CONTAINER_OPT" "$_arg_name" -- "${_arg_rule[@]}")
+command=(python3 "${script_dir}/automatus.py" rule ${ADDITIONAL_TEST_OPTIONS} --remove-platforms --remove-machine-only --remove-ocp4-only "${additional_args[@]}" "$CONTAINER_OPT" "$_arg_name" -- "${_arg_rule[@]}")
```

Comment:
Why do you add `--remove-machine-only` after `--remove-platforms`? From the help description it seems to me that `--remove-platforms` would remove all the platforms from everywhere, so I suppose it should remove also the `machine` platforms, but it means that `--remove-machine-only` isn't needed here. Am I correct? If I am wrong, can you change the help text of the `--remove-platforms` option to avoid confusions?

Reply:
The option `--remove-platforms` will only remove platforms from the Benchmark element and Profile element. It will not remove the machine platform that if I'm not mistaken can also reside under the Rule element. I will try to make things more clear